.github/workflows/pr-tests-e2e.yml

```diff
@@ -51,7 +51,56 @@ jobs:
       - uses: actions/upload-artifact@v4
         if: always()
         with:
-          name: playwright-app-report
+          name: playwright-app-report-stable
+          path: packages/app/playwright-results
+          retention-days: 30
+
+      - name: Stop Test Node
+        run: pnpm node:clean
+
+  tests-e2e-beta:
+    name: Test (Chrome Beta)
+    timeout-minutes: 20
```

Comment:
- Test times have decreased with the latest fixes, so we should try keeping this number at max 15 if possible

![CleanShot 2024-12-10 at 15 15 56](https://github.com/user-attachments/assets/2fe24549-6947-49cf-83f8-a350a2fbf2a2)